crates/matrix-sdk/src/client/mod.rs

```diff
@@ -344,6 +344,31 @@ impl Client {
         self.base_client().session_tokens().get_cloned()
     }
 
+    /// Get the current access token for this session.
+    ///
+    /// Will be `None` if the client has not been logged in.
+    ///
+    /// After login, this token should only change if support for [refreshing
+    /// access tokens] has been enabled.
+    ///
+    /// [refreshing access tokens]: https://spec.matrix.org/v1.3/client-server-api/#refreshing-access-tokens
+    pub fn access_token(&self) -> Option<String> {
+        self.session_tokens().map(|tokens| tokens.access_token)
+    }
+
+    /// Get the current refresh token for this session.
+    ///
+    /// Will be `None` if the client has not been logged in, or if the access
+    /// token doesn't expire.
+    ///
+    /// After login, this token should only change if support for [refreshing
+    /// access tokens] has been enabled.
+    ///
+    /// [refreshing access tokens]: https://spec.matrix.org/v1.3/client-server-api/#refreshing-access-tokens
```

Comment:
The refresh token doesn't change, just the access token, right?

Reply:
No, they both can change when you refresh your access token.

Reply:
Okay 👍